darts/models/rnn_model.py

```diff
@@ -145,12 +149,22 @@ def __init__(self,
             Sizes of hidden layers connecting the last hidden layer of the RNN module to the output, if any.
         dropout
             Fraction of neurons afected by Dropout.
+        random_state
+            Control the randomness of the weights initialization. Check this
+            `link <https://scikit-learn.org/stable/glossary.html#term-random-state>`_ for more details.
```

Comment:
Please correct me if I am wrong but I think that `random_state` from sklearn affects only function that is passed to. But here I see that torch seed will be set by `random_state` for all torch related pseudorandom number generation.

Reply:
yes you are absolutely right I didn't find a cleaner way to avoid side effect to you know any ?

Reply:
This looks good, It would need to be added also to fit function as well, I think we use shuffle=true in there, but it look possible with `fork_rng`. What do you think about just using `manual_seed` before using the model rather than in model itself? I feel like it would be much simpler.

Reply:
+1 for `manual_seed`

Reply:
> This looks good, It would need to be added also to fit function as well, I think we use shuffle=true in there, but it look possible with `fork_rng`. What do you think about just using `manual_seed` before using the model rather than in model itself? I feel like it would be much simpler.

it would be simpler indeed however we would not have a unified API meaning the models from sklearn would need to be passed a `random_state` while torch would need `torch.manual_seed` before usage for reproducibility